classes/class-p4-settings.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+if ( ! class_exists( 'P4_Settings' ) ) {
+	/**
+	 * Class P4_Settings
+	 */
+	class P4_Settings {
+
+		/**
+		 * Option key, and option page slug
+		 * @var string
+		 */
+		private $key = 'planet4_options';
+
+		/**
+		 * Array of metaboxes/fields
+		 * @var array
+		 */
+		protected $option_metabox = array();
+
+		/**
+		 * Options Page title
+		 * @var string
+		 */
+		protected $title = '';
+
+		/**
+		 * Options Page hook
+		 * @var string
+		 */
+		protected $options_page = '';
+
+		/**
+		 * Constructor
+		 */
+		public function __construct() {
+
+			// Set our title
+			$this->title   = __( 'Planet4', 'planet4-master-theme' );
+			$pages         = get_pages();
+
+			$pages_array[] = __( 'Select Page' ,'planet4-master-theme' ) ;
+			foreach ( $pages as $all_pages ) {
+				$pages_array[$all_pages->ID] = __( $all_pages->post_title ,'planet4-master-theme' ) ;
```

Comment:
spaces need fixing in this line

Reply:
Also, WP's translation functions expect string literals and not variables so that the translator can actually translate the string. Here you do not need to pass the post title to the translation function. The title will already be in the language that the editor wrote it.

Reply:
Yes, got it.